framework/src/postprocessors/FunctionValuePostprocessor.C

```diff
@@ -56,5 +76,11 @@ FunctionValuePostprocessor::execute()
 PostprocessorValue
 FunctionValuePostprocessor::getValue()
 {
-  return _scale_factor * _function.value(_t, _point);
+  Point p(0, 0, 0);
```

Comment:
```suggestion
  static const Point p;
```

as used in `Function.h`

Reply:
Oh god, this was a retarded comment....

Reply:
I think Daniel has changed his mind on this